napari/layers/labels/labels.py

```diff
@@ -48,6 +48,20 @@ class Labels(Image):
         Scale factors for the layer.
     translate : tuple of float
         Translation values for the layer.
+    rotate : float, 3-tuple of float, or n-D array.
+        If a float convert into a 2D rotation matrix using that value as an
+        angle. If 3-tuple convert into a 3D rotation matrix, rolling a yaw,
+        pitch, roll convention. Otherwise assume an nD rotation. Angle
+        conversion are done either using degrees or radians depending on the
+        degrees boolean parameter.
+    shear : 1-D array or n-D array
+        Either a vector of upper triangular values, or an nD shear matrix with
+        ones along the main diagonal.
+    affine: n-D array or napari.utils.transforms.Affine
+        (N+1, N+1) matrix where first (N, N) entries correspond to a linear
+        transform and the final column is a lenght N translation vector and
+        a 1 or a napari AffineTransform object. If provided then, scale,
+        rotate, and shear values are ignored.
```

Comment:
As above.

Holy moly you're right that we need to up our autogeneration game...

Reply:
lol